src/main/java/ua/softserveinc/tc/controller/admin/DayOffController.java

```diff
@@ -64,6 +68,7 @@
         if (dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
             log.warn("There is another day off with the same name: " + dayOff.getName() + ", or" +
                     "with the same start date: " + dayOff.getStartDate());
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
Let's see.
There was a method in controller, which checks if this dayOff already exists or no.No matter, if t exists or no, it wrote it to DB, but only shows the warning "log.warn("There is another day off with the same name: " + dayOff.getName() + ", or" +
                     "with the same start date: " + dayOff.getStartDate());"
This is not correct, because each time we create a similar dayOff it will be shown in Calendar as event. So, if there will be 10 similar dayoffs which starts in the same day and with same title, it will be the mess.